### PR TITLE
Projectkk2glider/lua bitmap free

### PR DESCRIPTION
**Pull request by projectkk2glider**
Result:

Before script:
```
meminfo
mallinfo:
	arena    1143808 bytes
	ordblks  126 bytes
	uordblks 1103632 bytes
	fordblks 40176 bytes
	keepcost 1096 bytes
Heap:
	start 0xd00bfc00
	end   0xd0800000
	curr  0xd01d7000
	used  1143808 bytes
	free  6459392 bytes
```

Script run and exited:
```
>Measure(storebackup) = 9523.5us
loaded 1
luaOpenBitmap: 0xd0186880
GC Use: 81925bytes
loaded 2
luaOpenBitmap: 0xd0107200
GC Use: 81962bytes
loaded 3
luaOpenBitmap: 0xd01ae9b8
GC Use: 82008bytes
loaded 4
luaOpenBitmap: 0xd01d0b20
GC Use: 82045bytes
loaded 5
luaOpenBitmap: 0xd0186930
GC Use: 82082bytes
loaded 6
luaOpenBitmap: 0xd0104e70
GC Use: 82128bytes
loaded 7
luaOpenBitmap: 0xd0105e38
GC Use: 82165bytes
loaded 8
luaOpenBitmap: 0xd01644b8
GC Use: 82202bytes
loaded 9
luaOpenBitmap: 0xd01abae8
GC Use: 82248bytes
loaded 10
luaOpenBitmap: 0xd010d720
GC Use: 82276bytes
loaded 11
luaOpenBitmap: 0xd010d758
GC Use: 82340bytes
loaded 12
luaOpenBitmap: 0xd01d0730
GC Use: 82368bytes
loaded 13
GC Use: 82336bytes
Script finished with status 2
luaDestroyBitmap: 0xd01d0730
luaDestroyBitmap: 0xd010d758
luaDestroyBitmap: 0xd010d720
luaDestroyBitmap: 0xd01abae8
luaDestroyBitmap: 0xd01644b8
luaDestroyBitmap: 0xd0105e38
luaDestroyBitmap: 0xd0104e70
luaDestroyBitmap: 0xd0186930
luaDestroyBitmap: 0xd01d0b20
luaDestroyBitmap: 0xd01ae9b8
luaDestroyBitmap: 0xd0107200
luaDestroyBitmap: 0xd0186880
GC Use: 79026bytes
GC Use: 78650bytes
```

After:
```
mallinfo:
	arena    1147904 bytes
	ordblks  135 bytes
	uordblks 1122552 bytes
	fordblks 25352 bytes
	keepcost 4008 bytes
Heap:
	start 0xd00bfc00
	end   0xd0800000
	curr  0xd01d8000
	used  1147904 bytes
	free  6455296 bytes
```

Some memory is not freed, it belongs to the `local` variables that are not set to nil in `cleanup()` function.

### CODE REVIEW

File: radio/sdcard/horus/S6R/S6R.lua [hunk 1]
```diff
@@ -310,8 +312,70 @@ local function init()
...
+    wingBitmaps[3] = Bitmap.open("img/planev_b.png")
+
+  elseif loaded == 4 then
+    mountBitmaps[1] = Bitmap.open("img/up.png")
+  elseif loaded == 5 then
+    mountBitmaps[2] = Bitmap.open("img/down.png")
+  elseif loaded == 6 then
+    mountBitmaps[3] = Bitmap.open("img/vert.png")
+  elseif loaded == 6 then
+    mountBitmaps[4] = Bitmap.open("img/vert-r.png")
```
Comment (bsongis) [inline radio/sdcard/horus/S6R/S6R.lua]: not sure we will enter here

**Comment by projectkk2glider** [comment]
I've made a big change - introduced new Lua state for the Themes and Widgets. 
* Lua Themes and Widgets moved to separate file and separate Lua state
* Stand-alone, mixer and function scripts now behave the same as on Taranis (restarted after the stand-alone script was run)
* both Lua states are independent: Themes and Widgets is initialized only at the start, the other one is initialized before and after the stand-alone script is run

This needs thorough testing before the merge! I only tested in simu.

**Comment by 3djc** [comment]
I'll do some testing tomorrow :)

**Comment by 3djc** [comment]
ok, I have run quite a few test:
- memory is freed for  standalone script, whenever they quit gracefully or are forced close
- permanent scripts keep running smoothly (tried with a custom mixer script)

Can't find any issue with this, only great improvements !

**Comment by bsongis** [comment]
Thanks!

**Pull request merged**